From 45939c652a16f23eb76d846ad7609a20a295a295 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Fri, 11 Sep 2020 20:41:39 +0300
Subject: [PATCH] [SW-17443] fit: goya2_arc: add public key property name

For accessing the public key by name and not by hard-coded path,
this patch saves public key and public key extension as property
in configuration node.

Change-Id: I0d32daf5a4caf6e73ea38f67a3fdf0c90bf8b769
---

diff --git a/board/habanalabs/common/uboot-zephyr.its b/board/habanalabs/common/uboot-zephyr.its
index b73021f..a70d13b 100644
--- a/board/habanalabs/common/uboot-zephyr.its
+++ b/board/habanalabs/common/uboot-zephyr.its
@@ -177,6 +177,8 @@
 			zephyr-pid-dccm = "zephyr-pid-dccm";
 			zephyr-mgmt-iccm = "zephyr-mgmt-iccm";
 			zephyr-mgmt-dccm = "zephyr-mgmt-dccm";
+			pub-key = "pub-key";
+			pub-key-ext = "pub-key-ext";
 			signature {
 				algo = "sha256,rsa3072";
 				key-name-hint = "hl_rsa3072";
